examples/ldns-keyfetcher.c

```diff
@@ -24,7 +24,7 @@ usage(FILE *fp, char *prog) {
 	fprintf(fp, "-4\t\tUse IPv4 only\n");
 	fprintf(fp, "-6\t\tUse IPv6 only\n");
 	fprintf(fp, "-h\t\tShow this help\n");
-	fprintf(fp, "-i\t\tInsecurer mode; don't do checks, just query for the keys\n");
+	fprintf(fp, "-i\t\tInsecure mode; don't do checks, just query for the keys\n");
```

Comment:
?

Reply:
In this case I believe the "Insecurer" is the intended word.